seccomp/seccomp.json

```diff
@@ -0,0 +1,1051 @@
+{
```

Comment:
This file should be created from the hosts seccomp.json file to make sure it is up to date.  It could either be created at install time or at least at package build time from the latest rhel version.

Reply:
thanks @rhatdan, going to prepare a second patch!